package.json

```diff
@@ -37,7 +37,7 @@
     "pelias-config": "2.1.0",
     "pelias-dbclient": "2.0.0",
     "pelias-logger": "0.0.8",
-    "pelias-model": "4.1.0",
+    "pelias-model": "4.2.0",
```

Comment:
The upgrade to 4.2.0 was [already merged](https://github.com/pelias/whosonfirst/pull/124). In general Greenkeeper always handles these quite quickly :)